Sources/Basic/OutputByteStream.swift

```diff
@@ -409,3 +488,25 @@ public struct Format {
         return TransformedSeparatedListStreamable(items: items, transform: transform, separator: separator)
     }
 }
+
+/// InMemory implementation of OutputByteStream.
```

Comment:
I hate how verbose it is, but I feel like we should probably use `BufferedOutputByteStream` for consistency.